dist/pathfora.js

```diff
@@ -562,7 +626,7 @@
      */
     parseQuery: function (url) {
       var query = {};
-      var pieces = url.split('?');
+      var pieces = utils.escapeURI(url, { keepEscaped: true }).split('?');
```

Comment:
By calling this function here, I believe we'd only be escaping the queries. Thus:

- `http://localhost:8080/examples/_bar.html?hello=hi there`
would match
- `http://localhost:8080/examples/_bar.html?hello=hi%20there`

but: 
- `http://localhost:8080/examples/_bar hi.html`
would not match 
- `http://localhost:8080/examples/_bar%20hi.html`

I might be wrong, but it seems like the desired functionality is to escape the whole url.